src/methodologies/para/detection/heuristics.py

```
@@ -127,17 +158,14 @@ def _contains_old_year(path_str: str, current_year: int, threshold_years: int =
 
     def evaluate(self, file_path: Path, metadata: dict[str, Any] | None = None) -> HeuristicResult:
         """Evaluate based on temporal patterns."""
-        import time
-        from datetime import UTC, datetime
-
         scores = {cat: CategoryScore(cat, 0.0, 0.0) for cat in PARACategory}
 
         if not file_path.exists():
             return HeuristicResult(scores, 0.0, None, True)
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)